components/storage/ComparisonCards.vue

```diff
@@ -16,19 +16,18 @@
       class="mx-3 my-3 px-3"
     >
       <template #header>
-        <h2 class="title is-size-4">{{ service.service | humanize }}</h2>
+        <h2 class="title is-size-4">{{ service.service }}</h2>
       </template>
 
       <template #content>
         <ul class="has-text-left">
           <li v-for="(feature, k) in service.features" :key="k">
-            <!-- {{ feature.name | humanize }} -->
             <div
               class="
                 is-flex is-justify-content-space-between is-align-items-center
               "
             >
-              <p>{{ feature.name | humanize }}</p>
+              <p>{{ feature.name }}</p>
```

Comment:
```suggestion
              <p>{{ humanize(feature.name) }}</p>
```
Unless no longer needed